src/components/character-select-entry.js

```diff
@@ -35,16 +35,18 @@ export default function CharacterSelectEntry({
       style={{
         flexDirection: "column",
         backgroundImage: `url(${image.src})`,
-        backgroundRepeat: "no-repeat"
+        backgroundRepeat: "no-repeat",
       }}
     >
       {/* Mobile */}
       <List.Content
         className="mobile-only"
         style={{
           position: "relative",
-          minHeight: "180px",
-          padding: "1rem"
+          padding: "1rem",
+          //Andrew made the following CSS rule changes
```

Comment:
We can see who wrote each line by using `git blame`, no need to comment it.

Reply:
awesome